client/src/partials/settings/settings.js

```diff
@@ -1,59 +1,62 @@
 angular.module('bhima.controllers')
-.controller('settings', [
-  '$scope',
-  '$routeParams',
-  '$translate',
-  '$location',
-  'appcache',
-  'messenger',
-  'tmhDynamicLocale',
-  'store',
-  'amMoment',
-  function($scope, $routeParams, $translate, $location, Appcache, messenger, tmhDynamicLocale, Store, amMoment) {
-    
-    // TODO issue discussing DB modelling of languages, quick suggestion (id, label, translateKey, localeKey)
-    var languageStore = new Store({identifier : 'translateKey'});
-    var languages = [
-      {
-        translateKey : 'en',
-        localeKey : 'en-us',
-        label : 'English'
-      },
-      {
-        translateKey : 'fr',
-        localeKey : 'fr-be',
-        label : 'French'
-      },
-      {
-        translateKey : 'ln',
-        localeKey : 'fr-cd',
-        label : 'Lingala'
-      }
-    ];
-    languageStore.setData(languages);
-    
-    $scope.url = $routeParams.url || '';
-    var cache = new Appcache('preferences');
+.controller('settings', Settings);
+
+Settings.$inject = ['$http','$routeParams','$translate','$location','appcache','messenger','tmhDynamicLocale','store','amMoment', 'SessionService'];
```

Comment:
Could you strip out unused injects?  Unused services like `messenger` might as well get removed now.